### PR TITLE
refactor: exofy Orchestrator, RemoteChainFacade

### PR DESCRIPTION
**Pull request by turadg**
refs: #9281

## Description
AsyncFlow requires that everything passing the membrane is durable. This makes the facade objects durable to conform.

Doing so for `localChainFacade` is deferred so we can get this into master sooner, to aid @erights 's https://github.com/Agoric/agoric-sdk/pull/9521/ .

### Security Considerations
none
### Scaling Considerations

Exo for each chain and each account

### Documentation Considerations
none
### Testing Considerations

Existing coverage

### Upgrade Considerations
none, not yet deployed

### CODE REVIEW

**Review by 0xpatrickdev — APPROVED** [review]
LGTM, we can polish the interface guards later

File: packages/orchestration/src/exos/orchestrator.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    'Orchestrator',
+    OrchestratorI,
+    () => {
+      trace('making an Orchestrator');
+      return {};
+    },
+    {
+      /** @type {Orchestrator['getChain']} */
+      getChain: async name => {
+        const agoricChainInfo = await chainHub.getChainInfo('agoric');
```
Comment (dtribble) [inline packages/orchestration/src/exos/orchestrator.js:76]: do these need to all be vow-based?

(I'm hoping not)

File: packages/orchestration/src/exos/remote-chain-facade.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+        const icaAccount = await V(orchestration).makeAccount(
+          remoteChainInfo.chainId,
+          connectionInfo.id,
+          connectionInfo.counterparty.connection_id,
+        );
+
+        const address = await V(icaAccount).getAddress();
+
+        const [{ denom: bondDenom }] = remoteChainInfo.stakingTokens || [
```
Comment (dtribble) [inline packages/orchestration/src/exos/remote-chain-facade.js:74]: comment this magic please

File: packages/orchestration/src/utils/start-helper.js [hunk 3]
```diff
@@ -59,12 +61,28 @@ export const provideOrchestration = (
     vowTools,
   });
 
+  const makeCosmosOrchestrationAccount = prepareCosmosOrchestrationAccount(
+    // XXX what zone?
```
Comment (dtribble) [inline packages/orchestration/src/utils/start-helper.js:65]: Add "FIXME"?

**Review by dtribble — APPROVED** [review]

**Comment by turadg** [comment]
I'll merge now in service of integration and follow up quickly to satisfy requests.

File: packages/orchestration/src/exos/orchestrator.js (hunk 1 above)
Comment (turadg) [reply packages/orchestration/src/exos/orchestrator.js:76]: No, they're awaiting at most a remote call to a synchronous method on agoricNames, so they will quiesce in any vat upgrade

**Pull request merged**